packages/web/hooks/fiat-getters.ts

```diff
@@ -0,0 +1,71 @@
+import { Dec, DecUtils, Int, PricePretty } from "@keplr-wallet/unit";
```

Comment:
I suggest we place this file within `packages/utils/src/swap.ts` as these methods will be useful for our Mobile app.

Reply:
I have attempted this but ran into the following issues:
- There is no dependency on the `server` package from `utils`, moving this would require adding it for the following:
`import { type Asset, DEFAULT_VS_CURRENCY } from "@osmosis-labs/server";`
- It just feels awkward since these are meant to be called from hooks only so hooks feels like a better location.

I've concluded to keep it as is based on the above two points but mainly because of the first one.

Thanks for recommending and let me know if there are further objections to this

Reply:
There seems to be some tech debt left after we created the `server` package. We can move both of these to the `utils` package. If you'd like, you can add these changes or I can create a PR. 

1) The `Asset` import should be a type called `MinimalAsset` over [`makeMinimalAsset`](https://github.com/osmosis-labs/osmosis-frontend/blob/59e4ed928dc53172e6ebc26cd58d5aeab04a3421/packages/utils/src/asset-utils.ts#L53). It would look like this 
```tsx
// In packages/types/src/asset-types.ts

export type MinimalAsset = Currency & {
  coinGeckoId: string | undefined;
  coinName: string;
  isUnstable: boolean;
  isVerified: boolean;
} 
```

```tsx
// In packages/utils/src/asset-utils.ts

import type { MinimalAsset } from "@osmosis-labs/types" 

/** Convert an asset list asset into an asset with minimal content and that
 *  is compliant with the `Currency` type. */
export function makeMinimalAsset(assetListAsset: Asset): MinimalAsset {
	...
}
```

2) Add a `packages/utils/src/currency.ts` file and export the `DEFAULT_VS_CURRENCY`


> It just feels awkward since these are meant to be called from hooks only so hooks feels like a better location.

Because these are standard TypeScript functions without reactivity, they can be invoked in any context, including on the server. Therefore, it's good to regard them as universal methods and store them in a utils folder. This approach also enhances their reusability across different platforms like mobile

I will approve this PR to not block these changes

Reply:
https://linear.app/osmosis/issue/FE-153/reduce-server-package-tech-debt